internal/security/bootstrapper/redis/config/config.go

```diff
@@ -20,11 +20,12 @@ import (
 )
 
 // ConfigurationStruct has a 1:1 relationship to the configuration.toml for the service. Writable is
-// the runtime extension of the static configuraiton.
+// the runtime extension of the static configuration.
 type ConfigurationStruct struct {
-	Writable    WritableInfo
-	SecretStore bootstrapConfig.SecretStoreInfo
-	Databases   map[string]bootstrapConfig.Database
+	Writable       WritableInfo
```

Comment:
Was going to mention this previously.  Seems strange that we have `Writable` when this service does use config provider. Can just have LogLevel at this level in the struct? 
Then GetLogLevel() returns `c.LogLevel` instead of `c.Writable.LogLevel`

Also below:

- UpdateFromRaw should do nothing as it will never be called 
- UpdateWritableFromRaw should do nothing as it will never be called
- EmptyWritablePtr should just return nil, also never called.

All this also applies to other config struct and TOML file.

Reply:
@lenny-intel 
this is beyond the scope of this PR

Reply:
I disagree for the new command, but if you prefer to do a follow up refactor PR for both, I am good with that.

Reply:
Yes, @lenny-intel , I would prefer doing them in another PR; would you please open an issue for that?

Reply:
Will do. I realized it is more than just security-bootstrapper that has this.

Reply:
Added. https://github.com/edgexfoundry/edgex-go/issues/3123